lib/determine-basal/determine-basal.js

```diff
@@ -147,7 +147,8 @@ var determine_basal = function determine_basal(glucose_status, currenttemp, iob_
         sens = profile.sens / autosens_data.ratio;
         sens = round(sens, 1);
         if (sens != profile.sens) {
-            console.error("Adjusting sens from "+profile.sens+" to "+sens);
+            process.stderr.write("sens from "+profile.sens+" to "+sens);
```

Comment:
What it ends up looking like is:
`Adjusting basal from 1.062 to 1.25; target_bg from 100 to 93; sens from 49.709 to 41.4`